index.js

```diff
@@ -0,0 +1,76 @@
+function sayHello(name) {
+    console.log("Hello, " + name + "!");{
+    if (name="Mark")
+    console.log("Hi, " + name + "!");}
+
+
+}
+  
+sayHello("Oleg");
+sayHello("Viktor");
+sayHello("Mark");
+
+const n = [3];
+const m = [4];
+console.log (( Math.pow(m, 2) ) + (Math.pow(n, 2)));
+
+console.log (Math.hypot(n, m));
+
+
+const a = (2)
+const b = (5)
+
+console.log (Math.min(a, b))
+
+
+const c = (3)
+const d = (-1)
+
+console.log (Math.min(c, d))
```

Comment:
You have to wrap it with function with 2 arguments and return the result from there, read task attentively